phoenix-core/src/main/java/org/apache/phoenix/calcite/CalciteUtils.java

```diff
@@ -1036,4 +1048,40 @@ public Void visitCall(RexCall call) {
             return null;
         }
     }
+
+    public static Object convertLiteral(SqlLiteral literal, PhoenixRelImplementor implementor) {
```

Comment:
Other convertXXX methods are to convert RexNode to Expression, so I'm thinking something like "convertSqlLiteral(...)" could be less confusing?